src/IBI.WZDx/Models/RoadEvents/WorkZones/DetourRoadEvent.cs

```diff
@@ -0,0 +1,194 @@
+/// </param>
+/// <param name="IsStartPositionVerified">
+/// Indicates if the start position (first geometric coordinate pair, see
+/// <see cref="RoadEventFeature.Geometry"/>) is based on actual reported  data from a GPS-equipped
+/// device that measured the location of the start of the detour.
+/// </param>
+/// <param name="IsEndPositionVerified">
+/// Indicates if the end position (last geometric coordinate pair, see
+/// <see cref="RoadEventFeature.Geometry"/>) is based on actual reported data from a GPS-equipped
+/// device that measured the location of the end of the detour.
+/// </param>
+/// <param name="Lanes">
+/// A list of individual lanes within a road event (roadway segment).
+/// </param>
+/// <param name="BeginningCrossStreet">
+/// Name or number of the nearest cross street along the roadway where the event begins.
+/// </param>
+/// <param name="EndingCrossStreet">
+/// Name or number of the nearest cross street along the roadway where the event ends.
+/// </param>
+/// <param name="BeginningMilepost">
+/// The linear distance measured against a milepost marker along a roadway where the event begins.
+/// </param>
+/// <param name="EndingMilepost">
+/// The linear distance measured against a milepost marker along a roadway where the event ends.
+/// </param>
+/// <param name="TypesOfWork">
+/// A list of the types of work being done in a road event and an indication of if each type
+/// results in an architectural change to the roadway.
+/// </param>
+/// <param name="WorkerPresence">
+/// Information about whether workers are present in the road event area.
+/// </param>
+/// <param name="ReducedSpeedLimitKph">
+/// The reduced speed limit posted within the road event, in kilometers per hour. This property only
+/// needs to be supplied if the speed limit within the road event is lower than the posted speed
+/// limit of the roadway.
+/// </param>
+/// <param name="Restrictions">
+/// A list of zero or more road restrictions that apply to the roadway segment described by this
+/// road event.
+/// </param>
+/// <param name="ImpactedCdsCurbZones">
+/// A list of references to external 
+/// <see href="https://github.com/openmobilityfoundation/curb-data-specification/tree/main/curbs#curb-zone">
+/// CDS Curb Zones
+/// </see> 
+/// impacted by the detour.
+/// </param>
+/// <param name="WorkZoneType">
+/// The type of detour road event, such as if the road event is static or actively moving as part
+/// of a moving operation.
+/// </param>
+public record DetourRoadEvent(
+    RoadEventCoreDetails CoreDetails,
+    DateTimeOffset StartDate,
+    DateTimeOffset EndDate,
+    LocationMethod LocationMethod,
+    VehicleImpact VehicleImpact,
+    bool? IsStartDateVerified = null,
+    bool? IsEndDateVerified = null,
+    bool? IsStartPositionVerified = null,
+    bool? IsEndPositionVerified = null,
+    WorkZoneType? WorkZoneType = null,
```

Comment:
Per the WZDx Specification, i don't think the DetourRoadEvent should have some of these properties. See https://github.com/usdot-jpo-ode/wzdx/blob/develop/spec-content/objects/DetourRoadEvent.md